contrib/test-suite/test-suite.yml

```diff
@@ -10,3 +10,46 @@
   - ci-build-debug/test-suite.log
   - ci-build-debug/ci-mock-result/*.log
   timeout: 6 hours
+
+- name: Multihost tests Tier 0
+  machines:
+  - client
+  tasks:
+  - name: Execute tests
+    shell: |
+      set -o nounset -o pipefail -o errexit -o xtrace
+
+      # Run only on RHEL based distributions
+      if [ ! -e /etc/redhat-release ]; then
```

Comment:
Btw, why do we skip non-RHEL8+?

Reply:
Downstream tests do not currently run multihost tests on older versions and it is not currently supported. The problem starts with package dependencies, maybe there more, I don't know. Given the state of RHEL-7 I don't think we should spend time on it.

Reply:
Just to prevent this .yml file getting messy.

Reply:
I mostly meant modern Fedora.

Reply:
Then I don't follow, AFAIK this file is available on Fedora.

Reply:
Ah, sorry, I was confused by `Run only on RHEL based distributions` comment

Reply:
Ok, it should be more clear now.